providers/catalog.go

```diff
@@ -1420,4 +1422,60 @@ var catalog = CatalogType{ // nolint:gochecknoglobals
 			Write:     false,
 		},
 	},
+
+	// InstagramBasic Support Configuration
+	InstagramBasic: {
+		AuthType: Oauth2,
+		BaseURL:  "https://graph.instagram.com",
+		OauthOpts: OauthOpts{
+			GrantType:                 AuthorizationCode,
+			AuthURL:                   "https://api.instagram.com/oauth/authorize",
+			TokenURL:                  "https://api.instagram.com/oauth/access_token",
+			ExplicitScopesRequired:    true,
+			ExplicitWorkspaceRequired: false,
+			TokenMetadataFields: TokenMetadataFields{
+				ScopesField: "scope",
+			},
+		},
+		Support: Support{
+			BulkWrite: BulkWriteSupport{
+				Insert: false,
+				Update: false,
+				Upsert: false,
+				Delete: false,
+			},
+			Proxy:     false,
+			Read:      false,
+			Subscribe: false,
+			Write:     false,
+		},
+	},
+
+	// InstagramGraphAPI Support Configuration
+	InstagramGraphAPI: {
+		AuthType: Oauth2,
+		BaseURL:  "https://graph.facebook.com",
```

Comment:
Can we call this `Facebook`? Seems like Instagram's professional API is built on top of facebook's graph API.

cc @laurenzlong

Reply:
Conflicts merged I have also updated the TokenMetadataFields to correct ones.

Instagram for professional users is based on Facebook Graph API, the difference is the additional functionality for instagram pages